lib/trino-plugin-toolkit/src/test/java/io/trino/plugin/base/security/TestFileBasedAccessControlConfig.java

```diff
@@ -102,4 +123,64 @@ public void testValidation()
                     .setConfigFile(securityConfigFile)
                     .setRefreshPeriod(Duration.valueOf("1ms")));
     }
+
+    @Test
+    public void testValidationWithUrl()
+    {
+        String securityConfigUrl = "http://test:1234/example";
+
+        assertFailsValidation(
+                new FileBasedAccessControlConfig()
+                        .setRefreshPeriod(Duration.valueOf("1ms")),
+                "configFile",
+                "may not be null",
+                NotNull.class);
+
+        assertFailsValidation(
+                new FileBasedAccessControlConfig()
+                        .setRefreshPeriod(Duration.valueOf("0ms"))
+                        .setConfigFile(securityConfigUrl),
+                "refreshPeriod",
+                "must be greater than or equal to 1ms",
+                MinDuration.class);
+
+        assertFailsValidation(
+                new FileBasedAccessControlConfig()
+                        .setConfigFile(securityConfigUrl)
+                        .setJsonPointer(null),
+                "jsonPointer",
+                "may not be null",
+                NotNull.class);
+
+        assertValidates(
+                new FileBasedAccessControlConfig()
+                        .setConfigFile(securityConfigUrl));
+
+        assertValidates(
+                new FileBasedAccessControlConfig()
+                        .setConfigFile(securityConfigUrl)
+                        .setRefreshPeriod(Duration.valueOf("1ms")));
+
+        assertValidates(
+                new FileBasedAccessControlConfig()
+                        .setConfigFile(securityConfigUrl)
+                        .setRefreshPeriod(Duration.valueOf("1ms"))
+                        .setJsonPointer("/data"));
+    }
```

Comment:
test for non-existing file ?

Reply:
it was already added in testValidationWithLocalFile